src/env.cc

```diff
@@ -559,7 +559,7 @@ void Environment::InitializeLibuv() {
   uv_prepare_init(event_loop(), &idle_prepare_handle_);
   uv_check_init(event_loop(), &idle_check_handle_);
 
-  uv_async_init(
+  CHECK_EQ(0, uv_async_init(
```

Comment:
There are a large number of `_init` calls in this function, isn’t it odd to single some out?

Reply:
@addaleax it's the only one that coverity reported as a problem.  Are you suggesting I should add them to the others as as well or not add at all?

Reply:
The difference might be that we check the return from uv_check_init everywhere else. This is the actual report:

```shell
CID 239987 (#1 of 1): Unchecked return value (CHECKED_RETURN)
3. check_return: Calling uv_async_init without checking return value (as is done elsewhere 15 out of 16 times).
```

Reply:
I see, that makes sense – Ideally, we’d check them everywhere, yes.

Reply:
The _init methods don't seem to use in other places.  I'll add to the other _init methods as well

Reply:
@addaleax added to the other methods in this function.